### PR TITLE
fix: adopt react library for lucide svg icons

### PR DESCRIPTION
**Pull request by rajarshisamaddar**
The Lucide react library icons have been implemented in this pull request, replacing all the existing SVG icons.

This resolves issue #205 regarding the adoption of the Lucide react library for SVG icons in the application.

Testing instructions:
1. To test, please ensure that all required packages specified in the package.json file are installed.
2. Then, thoroughly browse the web application to verify the icons are functioning properly.

### CODE REVIEW

**Review by use-to — CHANGES_REQUESTED** [review]
Our linter grumbles a bit here.
Use `pnpm run lint` locally. @rajarshisamaddar

**Comment by rajarshisamaddar** [comment]
> Our linter grumbles a bit here. Use `pnpm run lint` locally. @rajarshisamaddar

@use-to I'm really sorry I'm not aware of this thing can you please explain what is the problem maybe you can share some resources to learn from there about this?

**Comment by DasProffi** [comment]
> > Our linter grumbles a bit here. Use `pnpm run lint` locally. @rajarshisamaddar
> 
> @use-to I'm really sorry I'm not aware of this thing can you please explain what is the problem maybe you can share some resources to learn from there about this?

A linter is a tool/script that checks the formatting of the code. Running `pnpm run lint` outputs all lines that are malformed according to our code style. You have to apply changes to all lines that are marked with an error.

How it works can be found here: https://www.npmjs.com/package/eslint

But you only need to run the command locally and change the code lines marked with a lint error

File: tasks/components/OrganizationMemberList.tsx [hunk 1]
```diff
@@ -8,11 +8,11 @@ import { tw } from '@helpwave/common/twind/index'
 import type { Languages } from '@helpwave/common/hooks/useLanguage'
 import type { PropsWithLanguage } from '@helpwave/common/hooks/useTranslation'
 import { useTranslation } from '@helpwave/common/hooks/useTranslation'
-import Dropdown from '../icons/TriangleDown'
 import { Pagination } from './Pagination'
 import { Button } from './Button'
 import { Avatar } from './Avatar'
 import { Checkbox } from './user_input/Checkbox'
+import { ChevronDown as Dropdown } from 'lucide-react'
```
Comment (MaxSchaefer) [inline tasks/components/OrganizationMemberList.tsx:15]: We should avoid using `as`and should stick with the default naming of `lucide-react` to not confuse people in the future when they come across the icons.

**Review by MaxSchaefer — CHANGES_REQUESTED** [review]

**Comment by rajarshisamaddar** [comment]
@use-to @MaxSchaefer  all the request changes and checks are passed.

File: tasks/components/Pagination.tsx [hunk 2]
```diff
@@ -2,10 +2,10 @@ import { tw, tx } from '@helpwave/common/twind/index'
...
 import { useTranslation } from '@helpwave/common/hooks/useTranslation'
 import type { Languages } from '@helpwave/common/hooks/useLanguage'
-import ChevronLast from '@helpwave/common/icons/ChevronLast'
-import ChevronLeft from '@helpwave/common/icons/ChevronLeft'
-import ChevronFirst from '@helpwave/common/icons/ChevronFirst'
-import ChevronRight from '@helpwave/common/icons/ChevronRight'
+import { ChevronLast } from 'lucide-react'
+import { ChevronLeft } from 'lucide-react'
+import { ChevronFirst } from 'lucide-react'
+import { ChevronRight } from 'lucide-react'
```
Comment (DasProffi) [inline tasks/components/Pagination.tsx]:
```suggestion
import { ChevronLast, ChevronLeft, ChevronFirst, ChevronRight } from 'lucide-react'
```

**Review by DasProffi — CHANGES_REQUESTED** [review]

**Review by MaxSchaefer — APPROVED** [review]

**Comment by rajarshisamaddar** [comment]
@MaxSchaefer is there any task left to merge this pull request?

**Review by MaxSchaefer — APPROVED** [review]

**Comment by MaxSchaefer** [comment]
@use-to you requested changes. Can you revisit this pr please?

**Review by use-to — APPROVED** [review]

**Pull request merged**